Makefile

```diff
@@ -146,24 +146,26 @@ SPECTRAL ?= ${LOCAL_BIN_PATH}/spectral
 NPM ?= "$(shell which npm)"
 specinstall:
 ifeq (, $(shell which ${NPM} 2> /dev/null))
-	@echo "npm is not available please install it to be able to install spectral"
-	exit 1
+       @echo "npm is not available please install it to be able to install spectral"
+       exit 1
 endif
-ifeq (, $(shell which ${LOCAL_BIN_PATH}/spectral 2> /dev/null))
+ifeq (, $(shell which ${LOCAL_BIN_PATH}/rhoasapi 2> /dev/null))
 	@{ \
-	set -e ;\
-	mkdir -p ${LOCAL_BIN_PATH} ;\
-	mkdir -p ${LOCAL_BIN_PATH}/spectral-installation ;\
-	cd ${LOCAL_BIN_PATH} ;\
-	${NPM} install --prefix ${LOCAL_BIN_PATH}/spectral-installation @stoplight/spectral@5.9.2 ;\
-	${NPM} i --prefix ${LOCAL_BIN_PATH}/spectral-installation @rhoas/spectral-ruleset@0.1.4 ;\
-	ln -s spectral-installation/node_modules/.bin/spectral spectral ;\
+    set -e ;\
+    mkdir -p ${LOCAL_BIN_PATH} ;\
+    mkdir -p ${LOCAL_BIN_PATH}/rhoasapi-installation ;\
+	if [ -d ${LOCAL_BIN_PATH}/spectral-installation ]; then rm -r ${LOCAL_BIN_PATH}/spectral-installation; fi ;\
+	if [ -L ${LOCAL_BIN_PATH}/spectral ]; then rm ${LOCAL_BIN_PATH}/spectral; fi ;\
+     ${NPM} i --prefix ${LOCAL_BIN_PATH}/rhoasapi-installation @rhoas/spectral-ruleset@0.3.0-dev6 ;\
```

Comment:
This is using a dev ruleset `0.3.0-dev6`, what does that mean in terms of stability of the ruleset? Shouldn't we be using the stable-ish release instead?

Reply:
this will be very quickly bumped to a full release :) with very little changing. We decided to see how painful / painless updating all of the apis would be

Reply:
It'll be awesome to bump to a full release and not leave it a dev. Or have a followup issue where we track this to avoid forgetting about it ;-)